composer.json

```
@@ -7,7 +7,7 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": ">=7.4",
-        "google/auth": "^1.25.0",
+        "google/auth": "dev-add-universe-domain-phase2 as 1.31.0",
```

We can check against the main branch before the release
```suggestion
        "google/auth": "dev-main as 1.33.0",
```